.github/workflows/push-master-environment.yaml

```diff
@@ -78,13 +78,18 @@ jobs:
           HAPI_SERVER_PORT: 9090
           HAPI_POSTGRES_USER: ${{ secrets.HAPI_POSTGRES_USER }}
           HAPI_POSTGRES_PASSWORD: ${{ secrets.HAPI_POSTGRES_PASSWORD }}
+          HAPI_HASURA_URL: ${{ secrets.HAPI_HASURA_URL }}
+          HAPI_HASURA_ADMIN_SECRET: ${{ secrets.HAPI_HASURA_ADMIN_SECRET }}
           HAPI_PROXY_CONTRACT: regproxyinfo
           HAPI_RATING_CONTRACT: rateproducer
           HAPI_POSTGRES_DB: eosrate
           HAPI_POSTGRES_HOST: postgres
           HAPI_POSTGRES_PORT: 5432
           HAPI_POSTGRES_SCHEMA: public
           HAPI_EOS_API_ENDPOINT: https://api.main.alohaeos.com
+          HAPI_EOS_CHAIN_ID: aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906
```

Comment:
this is the chain ID for jungle 3

`2a02a0053e5a8cf73a56ba0fda11e4d92e0238a4a2aa74fccf46d5a910746840`